### ISSUE TITLE
PlayMusic.init does not have error callback

### ISSUE THREAD

**Issue by wyozi** [jamon/playmusic#8]
https://github.com/jamon/playmusic/blob/master/play.js#L113 refers to an inexistent argument ```error```, so it is not possible to have a error callback for the login.

**Comment by rlemon** [comment]
Just piggy backing on this, 

`````
        pm.init({
            email: ...,
            password: ...
        }, successHandler, function(err) {
            console.log('here'); 
        });
`````

entering incorrect credentials results in console.error spitting out `[Function] '\u001b[32m\'Error=BadAuthentication\\n\'\u001b[39m'` and the error callback I provide is never hit. 

Am I doing something wrong here?
using playmusic@1.1.1

**Comment by jamon** [comment]
try doing an npm install jamon/playmusic and seeing if that resolves the issue.  If so, I'll push an update to npm.

Thanks,
Jamon

**Comment by rlemon** [comment]
I apologize, I don't understand what you want me to run, are you saying use the code on github vs installing from npm?

**Comment by jamon** [comment]
Yes.  npm has a shorthand for installing directly from github, you can just do "npm install user/repo"

**Comment by rlemon** [comment]
okay, installing from jamon/playmusic had no change. The callback is still never fired and console.error displays. 

I also have tried downloading play.js and running from that but it crashes on bad creds. 

`````
/home/rlemon/Development/gm-cli/PlayMusic.js:100
        that._token = response.Auth;
                              ^
TypeError: Cannot read property 'Auth' of undefined
    at /home/rlemon/Development/gm-cli/PlayMusic.js:100:31
    at Object.request.error (/home/rlemon/Development/gm-cli/PlayMusic.js:146:13)
    at IncomingMessage.<anonymous> (/home/rlemon/Development/gm-cli/PlayMusic.js:71:25)
    at IncomingMessage.emit (events.js:117:20)
    at _stream_readable.js:944:16
    at process._tickCallback (node.js:442:13)
`````

**Comment by jamon** [comment]
I'm seeing the same thing.

**Comment by jamon** [comment]
This should be fixed in 2.0, though the error handling/callbacks are slightly different.  let me know if you're still having issues.
